templates/cjs/typescript_gapic/tsconfig.json.njk

```diff
@@ -28,6 +28,8 @@ limitations under the License.
     "src/**/*.ts",
     "test/*.ts",
     "test/**/*.ts",
-    "system-test/*.ts"
+    "system-test/*.ts",
+    "src/**/*.json",
+    "protos/protos.json"
```

Comment:
```suggestion
    "samples/**/*.json",
    "protos/protos.json"
```

Reply:
Samples shouldn't be included in ts linting, so I think better to add samples to eslintignore (if it's not already there!)

Reply:
Duh, makes sense!